packages/core/src/cli.ts

```diff
@@ -95,20 +95,21 @@ export class CommandLineInterface {
      * @returns {Contracts.CommandList}
      * @memberof CommandLineInterface
      */
-    private discoverCommands(): Contracts.CommandList {
+    private discoverCommands(dirname: string): Contracts.CommandList {
         const discoverer = this.app.resolve(Commands.DiscoverCommands);
+        const commands: Contracts.CommandList = discoverer.within(resolve(dirname, "./commands"));
 
-        const commands: Contracts.CommandList = discoverer.within(resolve(__dirname, "./commands"));
         const commandsFromPlugins = discoverer.from(
             this.app.get<Contracts.Config>(Container.Identifiers.Config).get("plugins"),
         );
 
         for (const [key, value] of Object.entries(commandsFromPlugins)) {
+            // TODO: check we can ignore this line for now
```

Comment:
Can be ignored for now, I'll resolve it once I finalise the implementation of the CLI.